defaults/main.yml

```diff
@@ -32,15 +35,23 @@ bind9_hidden_master: no
 # Necessary to keep traffic between nameservers in private network.
 bind9_notify_explicit: no
 
+# bind9_notify: '{{ "explicit" if bind9_notify_explicit else undef }}'
+# undef doesn't work here. f**k legacy bind9_notify_explicit variable?
+
```

Comment:
I think this comment not necessary to merge, since it's not used

Reply:
I agree. And it's not very nice for the author of this variable... Sorry.I was supposed to be development temporary notes.